zephyr/rz/rz_cfg/fsp_cfg/rzg/r_riic_master_cfg.h

```diff
@@ -0,0 +1,19 @@
+/*
+* Copyright (c) 2020 - 2024 Renesas Electronics Corporation and/or its affiliates
+*
+* SPDX-License-Identifier: BSD-3-Clause
+*/
+#ifndef R_RIIC_MASTER_CFG_H_
+#define R_RIIC_MASTER_CFG_H_
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define RIIC_MASTER_CFG_PARAM_CHECKING_ENABLE (BSP_CFG_PARAM_CHECKING_ENABLE)
+#define RIIC_MASTER_CFG_DMAC_ENABLE (0)
+#define RIIC_MASTER_CFG_ADDR_MODE_10_BIT_ENABLE (0)

```

Comment:
@nhutnguyenkc , is it for now I2C driver of RZ will not support the 10 bit addressing mode?

Reply:
@duynguyenxa , thank you, I have enabled 10-bit addressing mode, and already passed i2c test.